From: Debian DM Multipath Team <team+linux-blocks@tracker.debian.org>
Date: Wed, 23 Dec 2020 22:52:02 +0000
Subject: systemd: alias multipath-service

Add Alias of multipath-tools because our init script is named that way.
---
 multipathd/multipathd.service | 1 +
 1 file changed, 1 insertion(+)

diff --git a/multipathd/multipathd.service b/multipathd/multipathd.service
index 1919b38..0e60fb8 100644
--- a/multipathd/multipathd.service
+++ b/multipathd/multipathd.service
@@ -23,3 +23,4 @@ TasksMax=infinity
 [Install]
 WantedBy=sysinit.target
 Also=multipathd.socket
+Alias=multipath-tools.service
